### PR TITLE
Improve test structure

### PR DESCRIPTION
**Pull request by lukeperrin10**
pt story : https://www.pivotaltracker.com/story/show/179056810

## Suggested changes 

- updates test files for testing of multiple view-ports in same test than separate ones
- removes un-needed extra file structure
- 25 files removed, test files now stand at 10

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -14,7 +14,16 @@
...
-import './commands'
+import "./commands";
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
+
+const sizes = [
+  "iphone-x",
+  ["iphone-x", "landscape"],
+  ["ipad-2", "landscape"],
```
Comment (1CIM) [inline cypress/support/index.js:25]: ain't ipad 2 landscape pretty much the same as macbook? maybee better with just ipad 2 no landscape?

**Review by 1CIM — COMMENTED** [review]
Nice getting rid of the test hole 👍

File: cypress/support/index.js (hunk 1 above)
Comment (lukeperrin10) [reply cypress/support/index.js:25]: Yeah ok, kinda close, sure I can update it.

Comment (lukeperrin10) [reply cypress/support/index.js]: Fixed, had to remove some switch statements for the portrait version

**Review by 1CIM — APPROVED** [review]
LGTM 👍

**Pull request merged**